WooCommerce/Classes/ViewRelated/Products/Cells/ProductReviewsTableViewCell.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+import Gridicons
+
+
+// MARK: - ProductReviewsTableViewCell
+//
+class ProductReviewsTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var reviewLabel: UILabel!
+    @IBOutlet weak var reviewTotalsLabel: UILabel!
+    @IBOutlet weak var starRatingView: RatingView!
+
+    var starRating: Int? {
+        didSet {
+            guard let starRating = starRating else {
+                starRatingView.isHidden = true
+                return
+            }
+
+            starRatingView.rating = CGFloat(starRating)
+        }
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        reviewLabel.applyBodyStyle()
+        reviewTotalsLabel.applyBodyStyle()
+        configureStarView()
+    }
+
+    func configureStarView() {
+        starRatingView.starImage = Star.filledImage
+        starRatingView.emptyStarImage = Star.emptyImage
+    }
+}
+
+
+// MARK: - Constants
+//
+private extension ProductReviewsTableViewCell {
+    enum Star {
+        static let size = Double(20)
```

Comment:
Would it make sense to declare this directly as a CGSize? it would make the other two cases shorter...

Reply:
Good point! I absent-mindedly copy-pasted this code from somewhere else because I've never implemented the review stars before. Will fix :)